examples/ads.amp.html

```diff
@@ -287,6 +288,15 @@ <h2 id="aduptech">Ad Up Technology</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="adverline">Adverline</h2>
+  <amp-ad width=300 height=250
+          type="adverline"
```

Comment:
please indent 4 spaces.
could you also fix the following example for AdvertServce?